ec/src/msm/mod.rs

```diff
@@ -3,6 +3,8 @@ mod variable_base;
 pub use fixed_base::*;
 pub use variable_base::*;
 
+pub mod stream_pippenger;
```

Comment:
I would move this into `variable_base`, since it's a variable-base MSM

Reply:
This is the core internal algorithm of streaming MSM. But we do have another streaming MSM function for public use that has not been imported yet because it relies on streaming utilities. The plan is to first decide which repo to put streaming utilities and then import streaming MSM into variable-base MSM.

Reply:
Pratyush, I think here involves a decision, that is where to put this newer stream variable-base MSM---it clearly belongs to variable base. Does it replace the original variable base algorithm? If not, how should we name then?

Reply:
I mean that we add this algorithm as another file in the `variable_base` module, instead of making it a top-level member of the `msm` module.